Application.groovy

```diff
@@ -1,7 +1,6 @@
-@Grab("edu.wisc.my.restproxy:json-proxy-service:1.0")
+@Grab("edu.wisc.my.restproxy:json-proxy-service:1.1.6-SNAPSHOT")
```

Comment:
snapshots are deployed to central's snapshot repo, but you are right, its the wild wild west.